starknet/src/space/space.cairo

```diff
@@ -5,28 +5,71 @@ use sx::types::{
 
 #[starknet::interface]
 trait ISpace<TContractState> {
-    // State 
+    // -- View functions --
+
+    /// Returns the owner of the contract.
     fn owner(self: @TContractState) -> ContractAddress;
-    fn max_voting_duration(self: @TContractState) -> u32;
+
+    /// The voting delay to wait for between a proposal is created and the start of the voting period.
+    fn voting_delay(self: @TContractState) -> u32;
+
+    /// Returns the minimum voting duration. Once this duration has elapsed,
+    /// a proposal can be finalized. Most DAOs should set `min_voting_duration` and
+    /// `max_voting_duration` to the same value.
     fn min_voting_duration(self: @TContractState) -> u32;
+
+    /// Returns the maximum voting duration. Once this duration has elasped, a proposal
+    /// can be finalized. No other votes can come in.
```

Comment:
Once this duration has elapsed, no additional votes can be cast.

Reply:
3b561eb2d3c5ee7376822f546c0fe081368ddb63

Reply:
90a90ccc7f75db0456bd19d7cace99868b8f3f57